sdk/computefleet/azure-resourcemanager-computefleet/src/test/java/com/azure/resourcemanager/computefleet/ComputeFleetManagerTests.java

```diff
@@ -0,0 +1,291 @@
+import com.azure.resourcemanager.computefleet.models.VirtualMachineScaleSetManagedDiskParameters;
+import com.azure.resourcemanager.computefleet.models.VirtualMachineScaleSetNetworkConfiguration;
+import com.azure.resourcemanager.computefleet.models.VirtualMachineScaleSetNetworkConfigurationProperties;
+import com.azure.resourcemanager.computefleet.models.VirtualMachineScaleSetNetworkProfile;
+import com.azure.resourcemanager.computefleet.models.VirtualMachineScaleSetOSDisk;
+import com.azure.resourcemanager.computefleet.models.VirtualMachineScaleSetOSProfile;
+import com.azure.resourcemanager.computefleet.models.VirtualMachineScaleSetStorageProfile;
+import com.azure.resourcemanager.computefleet.models.VmSizeProfile;
+import com.azure.resourcemanager.network.NetworkManager;
+import com.azure.resourcemanager.network.models.LoadBalancer;
+import com.azure.resourcemanager.network.models.LoadBalancerSkuType;
+import com.azure.resourcemanager.network.models.Network;
+import com.azure.resourcemanager.network.models.TransportProtocol;
+import com.azure.resourcemanager.resources.ResourceManager;
+import com.azure.resourcemanager.resources.fluentcore.utils.ResourceManagerUtils;
+import com.azure.resourcemanager.resources.models.Provider;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Random;
+import java.util.UUID;
+
+public class ComputeFleetManagerTests extends TestProxyTestBase {
+    private static final Random RANDOM = new Random();
+    private static final Region REGION = Region.US_WEST2;
+    private String resourceGroupName = "rg" + randomPadding();
+    private ComputeFleetManager computeFleetManager = null;
+    private NetworkManager networkManager = null;
+    private ResourceManager resourceManager;
+    private boolean testEnv;
+
+    @Override
+    public void beforeTest() {
+        final TokenCredential credential = new DefaultAzureCredentialBuilder().build();
+        final AzureProfile profile = new AzureProfile(AzureEnvironment.AZURE);
+
+        computeFleetManager = ComputeFleetManager
+            .configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BODY_AND_HEADERS))
+            .authenticate(credential, profile);
+
+        networkManager = NetworkManager
+            .configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BODY_AND_HEADERS))
+            .authenticate(credential, profile);
+
+        resourceManager = ResourceManager
+            .configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BODY_AND_HEADERS))
+            .authenticate(credential, profile)
+            .withDefaultSubscription();
+
+        canRegisterProviders(Arrays.asList("Microsoft.Resources", "Microsoft.Network", "Microsoft.AzureFleet"));
```

Comment:
I don't think you need to add Resources and Network. They should always be there.

Reply:
Fixed in the new version.